capstone-overview/advice-from-staff.md

```diff
@@ -0,0 +1,25 @@
+# Advice from Staff
+
+This is just some additional advice from staff to supplement but certainly not replace the [Advice from Students](advice-from-students.md) document.
+
+
+## Sprint Board
+
+Tools are more useful when we understand their purpose. A planning (Trello) board is meant to help you with:
+* Focus (ideally one thing in progress at a time) 
+    * Also when you think of other things you need to do, you can add a card to the backlog real quick and come back to your current task.
+* A sense of completion (hopefully your cards are small enough that you’re able to complete at least one almost every day)
+* A visual representation of your priorities - ensuring you will get to the things that are most important and not get distracted by other things
+    * Organize Trello cards by “Sprint/Week”. (This way you have some idea how you’re tracking: whether you’re ahead, behind, or on-track)
+* Coordination - understanding who is working on what (for pairs only)
+
+## Standup
+
+It is very common for standup meetings to talk much more time than anyone really wants them to. It may help to write down your answers to the 3 standup questions before standup time or make time for writing answers down at the beginning of standup time. (This helps to avoid rambling or forgetting something important
+
```

Comment:
```suggestion

The three questions to answer at each standup are:
* What did you do since last standup?
* What are you going to work on next?
* Do you have any blockers (things you need help with)?
```